.github/workflows/azure-functions-app-dotnet.yml

```
@@ -18,45 +1,327 @@
+          }
+      
+      - name: Deploy to Azure Functions
+        if: steps.check-project.outputs.project-exists == 'true'
+        uses: Azure/functions-action@v1
+        id: deploy
+        with:
+          app-name: ${{ needs.validate.outputs.function-app-name }}
+          package: '${{ env.AZURE_FUNCTIONAPP_PACKAGE_PATH }}/output'
+          publish-profile: ${{ secrets.AZURE_FUNCTIONAPP_PUBLISH_PROFILE }}
+          respect-funcignore: true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)